src/decode.rs

```diff
@@ -2895,36 +2904,38 @@ unsafe fn decode_b_inner(
             } else {
                 has_subpel_filter = true;
                 *b.inter_mode_mut() = NEWMV;
-                *b.drl_idx_mut() = NEAREST_DRL;
+                *b.drl_idx_mut() = DrlProximity::Nearest;
                 if n_mvs > 1 {
                     // NEARER, NEAR or NEARISH
```

Comment:
this comment is misleading, because `NEARISH` can never actually be assigned here

Reply:
True, you can fix the comment.